cluster-autoscaler/core/scale_down.go

```diff
@@ -1065,7 +1069,7 @@ func (sd *ScaleDown) scheduleDeleteEmptyNodes(emptyNodes []*apiv1.Node, client k
 			return deletedNodes, errors.ToAutoscalerError(errors.ApiCallError, taintErr)
 		}
 		deletedNodes = append(deletedNodes, node)
-		go func(nodeToDelete *apiv1.Node, nodeGroupForDeletedNode cloudprovider.NodeGroup) {
+		go func(nodeToDelete *apiv1.Node, nodeGroupForDeletedNode cloudprovider.NodeGroup, evictDaemonSetPodsBool bool) {
```

Comment:
You don't need to include bool in the name, since it's implied by the type.

Reply:
I already have evictDaemonSetPods function

Reply:
evictDsPods?

Reply:
I find existed name more explicit, otherwise we will have quite similar name for bool value and function name

Reply:
Adding type information to variable names is generally discouraged in go, so I'd really like to see a different name here, but it's not super important.